### DISCUSSION TITLE
Enable Dependabot on upptime/uptime-monitor action and keep us all safer

### DISCUSSION THREAD

**Discussion by rajbos** [upptime/upptime#639] [Q&A]
We want to onboard this project and always run a security scan on the action. One of the steps is forking the action and enabling Dependabot to see what it finds.

In this case it found a lot of vulnerable dependencies 😢. Could you enable Dependabot and let it propose fixes for the vulnerable dependencies it found? It's free for public repos and will keep your dependencies up to date if you let it.

![image](https://user-images.githubusercontent.com/6085745/185413933-4799fee9-c064-477a-b6dd-b4049bd879d3.png)

Happy to help if needed.

#### Note: moved this over based on the initial PR [here](https://github.com/upptime/uptime-monitor/pull/187)

**Comment by AnandChowdhary** [comment]
For sure! Would you be up for a PR to update dependencies to a non-vulnerable version?

**Reply by rajbos** [reply]
Easiest way to do so is to enable Dependabot on the repo and have it propose updates itself 😉

**Reply by rajbos** [reply]
Go to the repo settings --> Code Security and analysis and enable these to features (for free!):

![image](https://user-images.githubusercontent.com/6085745/185624390-99177483-c064-4016-a0da-9754654f2aec.png)

Whenever Dependabot knows about a vulnerable dependency, it will send you a PR for that dependency.

This will make sure this always happens, and not only when a user sends in a notification.
